RockWeb/Themes/NewSpring/assets/Lava/DATASETS/content-channel-items-by-channel.lava

```diff
@@ -1,6 +1,6 @@
 {%- if ccid != empty -%}
 [
-    {% contentchannelitem dataview:'3552' where:'ContentChannelId == {{ ccid }}' sort:'StartDateTime desc' iterator:'items' -%}
+    {% contentchannelitem where:'ContentChannelId == {{ ccid }} && Status != 3' limit:'10000' sort:'StartDateTime desc' iterator:'items' -%}
```

Comment:
Here, we're removing the dataset limitation to include future-dated and expired content (to be filtered out later for unauthorized users). We're omitting items with a status of `Denied`, and statically set a limit of 10,000 entries as not specifying a limit defaults to limiting to 1,000 - which we have more than.